source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -54,82 +54,83 @@ The following figure depicts the import process of an OVA that was generated by
-4. The front-end sends (an optionally customized) parameters of the import process to the back-end.
-5. The back-end persists the VM.
-6. The back-end triggers the creation of the target VM disks on the storage domain.
-7. The back-end sends the import parameters, including the destination disks, to imageio-proxy, that sends them to imageio.  
+1. The front end sends the back end a VM and a path on a host.  
+2. The back end retrieves the OVF configuration from the OVA using an agent located on the host (Imageio/Other).  
+3. The back end parses the OVF configuration and returns a VM to the front end.
+4. The front end sends (optionally customized) parameters of the import process to the back end.
+5. The back end persists the VM into the database.
+6. The back end triggers the creation of the target VM disks on the storage domain.
+7. The back end sends the import parameters, including the destination disks, to imageio-proxy, that sends them to imageio.  
 8. Imageio reads the disk from the OVA and according to the import parameters streams the data into the corresponding target disks.
 
 ## Downloading an OVA
-This part depends on the implementation of the download-image flow. In theory, it should look like that:  
+This part depends on the implementation of the download-image flow. In theory, it should look like this:  
 
 ![](../../../../images/wiki/download-ova.png)  
 
-1. The front-end passes the back-end a VM (or template) to download.
-2. The back-end sends import parameters to imageio-proxy in order to prepare to download-ova.
-3. The URL that is returned by imageio-proxy is returned to the front-end through the back-end.
-4. The front-end initialize a target file to write the OVA to and starts listening on the URL.
-5. The back-end passes imageio-proxy the disk to copy and imageio-proxy delivers it to imageio.
-6. Imageio starts reading the disk, converting it to QCOW and sends it to the URL. The front-end then streams the data into the target file.
-7. The back-end sends imageio-proxy the OVF configuration, which sends it to imageio that writes it to the URL. The front-end then streams the data into the target file.
+1. The front end passes the back end a VM (or template) to download.
+2. The back end sends import parameters to imageio-proxy in order to prepare to download-ova.
+3. The URL that is returned by imageio-proxy is returned to the front end through the back end.
+4. The front end initialize a target file to write the OVA to and starts listening on the URL.
+5. The back end passes imageio-proxy the disk to copy and imageio-proxy delivers it to imageio.
+6. Imageio starts reading the disk, converting it to QCOW and sends it to the URL. The front end then streams the data into the target file.
+7. The back end sends imageio-proxy the OVF configuration, which sends it to imageio that writes it to the URL. The front end then streams the data into the target file.
```

Comment:
@ahadas Suggest "The back end sends the OVF configuration to imageio-proxy. imageio-proxy sends it to imageio, and imageio writes the OVF configuration to the URL."